src/hooks/useAccountBalance.ts

```diff
@@ -93,15 +93,15 @@ export const useAccountBalance = ({
   const { formatted: evmBalance } = evmQuery.data || {};
   const balance = !assetSymbol ? '0' : isCosmosChain ? cosmosQuery.data : evmBalance;
 
-  const nativeTokenCoinBalance = balances?.[DYDX_DENOM];
+  const nativeTokenCoinBalance = balances?.[chainDenom];
   const nativeTokenBalance = MustBigNumber(nativeTokenCoinBalance?.amount)
     .div(QUANTUM_MULTIPLIER)
```

Comment:
is this still accurate? or do we need to shift it by the perspective decimals?

Reply:
oh nice catch